terraform/orb.yml

```diff
@@ -211,7 +211,11 @@ commands:
           name: "terraform workspace new"
           command: |
             include init.sh
-            exec terraform workspace new "$workspace" "$module_path"
+            exec if [[ -z "$(terraform workspace list $module_path | grep $workspace)" ]]; then \
+              terraform workspace new "$workspace" "$module_path"; \
```

Comment:
Can you `exec terraform`, not the if. Doesn't matter much here, but for consistency

Reply:
Done :D